%YAML:1.0

hexPosCtrl:
  HEX_THR_MIN: 0.12f
  HEX_THR_MAX: 1.0f
  HEX_THR_HOVER: 0.5f
  HEX_THR_CURVE: 0
  HEX_HRI_P: 0.55
  HEX_Z_VEL_P: 0.2f
  HEX_Z_VEL_I: 0.02f
  HEX_Z_VEL_D: 0.0f
  HEX_XY_VEL_P: 0.09f
  HEX_XY_VEL_I: 0.02f
  HEX_XY_VEL_D: 0.01f


